src/commons/buttons/index.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
```

Comment:
Hey @wasibani-roy ,This file has some linting errors,you could run `npm fix` or you could write it as 
```
import React from "react";

export default function Button({
  text, onClick, type, isProcessing, className, id
}) {
  return (
    <button
      type={type}
      disabled={isProcessing}
      id={id}
      onClick={onClick}
      className={className}
    >
      {text}
    </button>
  )
}

```